src/alpine/.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
Can we update `versions` for each of the Templates so that these changes get published?

Reply:
updated!